examples/rbd/storageclass.yaml

```diff
@@ -24,6 +24,10 @@ parameters:
    # CSI RBD currently supports only `layering` feature.
    imageFeatures: layering
 
+   # With volumenameprefix option an admin can now prefix the desired volume
+   # name from storageclass.
+   # volumenameprefix: custom-vol-name
```

Comment:
Nit: seems this should be camelCase?

Reply:
@dillaman this is to support feature parity with OCS 3.x. if it's accepted I can  change it

Reply:
It's a new storage class so there is zero need to be binary compatible w/ old naming.

Reply:
... i.e. feature parity does not mean exactly duplicated implementation (for obvious reasons)